spec/index.html

```diff
@@ -734,9 +732,9 @@ <h3>Translation to the SPARQL Algebra</h3>
             </pre>
       </section>
       <section id="algebra_service">
-        <h3>SPARQL 1.1 Simple Federation Extension Algebra</h3>
-        <p>The evaluation of <code>SERVICE</code> is defined in terms of the <a class="inform" href="http://www.w3.org/TR/rdf-sparql-XMLres/">SPARQL Results</a> [<a href="#RESULTS">RESULTS</a>]
-        returned by a SPARQL Protocol [<a href="#SPROT">SPROT</a>] execution of the nested graph pattern:</p>
+        <h3>SPARQL Simple Federation Extension Algebra</h3>
+        <p>The evaluation of <code>SERVICE</code> is defined in terms of the [[[SPARQL12-RESULTS-XML]]]
```

Comment:
XML results link is informative:

```suggestion
        <p>The evaluation of <code>SERVICE</code> is defined in terms of the [[[?SPARQL12-RESULTS-XML]]]
```